projects/react/src/lib/useMaskito.ts

```diff
@@ -32,26 +33,32 @@ export const useMaskito = ({
     options?: MaskitoOptions;
     elementPredicate?: MaskitoElementPredicate;
 } = {}): RefCallback<HTMLElement> => {
-    const [element, setElement] = useState<HTMLElement | null>(null);
+    const [predicateResult, setPredicateResult] = useState<
+        HTMLInputElement | HTMLTextAreaElement | null
+    >(null);
 
     const onRefChange: RefCallback<HTMLElement> = useCallback(
-        (node: HTMLElement | null) => {
-            setElement(node);
+        async (node: HTMLElement | null) => {
+            if (node) {
+                const predicate = await elementPredicate(node);
```

Comment:
@sean-perkins 
Could you add new unit tests (to test async predicate) for `@maskito/react` package, please ?

https://github.com/Tinkoff/maskito/blob/5783e766a657abcf0fc7f8a8d12ac1bf412dc18a/projects/react/src/lib/useMaskito.spec.tsx#L22